webapp/src/components/news-item/NewsItem.stories.js

```diff
@@ -0,0 +1,23 @@
+import Vue from 'vue';
+import Vuetify from 'vuetify';
+import NewsItem from "./NewsItem.vue";
+
+Vue.use(Vuetify);
+
+export default {
+    title: 'Components/NewsItem',
+    component: NewsItem,  
+    argTypes: {
```

Comment:
```suggestion
    argTypes: {
      upvote: { action: 'upvote' },
```

And you can do 
```
    template: '<news-item :newsItem="newsItem" @upvote="upvote" />',
```
on line 18 to set up event handlers for your events in your storybook.